src/SWP/Bundle/ContentBundle/Resources/config/services.yml

```diff
@@ -5,6 +5,7 @@ parameters:
     swp_content_bundle.listener.http_cache_header.class: SWP\Bundle\ContentBundle\EventListener\HttpCacheHeaderListener
     swp_content_bundle.manager.media.class: SWP\Bundle\ContentBundle\Manager\MediaManager
     swp_content_bundle.router.media.class: SWP\Bundle\ContentBundle\Routing\MediaRouter
+    swp.redirecting.notfound.section: '%env(bool:REDIRECT_NOT_FOUND_ARTICLES_TO_ROUTE)%'
```

Comment:
It would be nice to have this configurable via UI in the tenant's settings or tenant object instead of env var